packages/flutter/lib/src/material/dialog.dart

```diff
@@ -494,6 +518,15 @@ class SimpleDialog extends StatelessWidget {
   ///    value is used.
   final String semanticLabel;
 
+  /// The shape of this [AlertDialog]'s border.
```

Comment:
[SimpleDialog]

Reply:
Good catch, now obsolete though (updated this comment to use @macro.